docs/index.md

```diff
@@ -118,6 +124,16 @@ invoking `mpirun`. After running the previous command, you should also run the
 tests with at least 4 nodes. To learn how to run the MPI tests on multiple
 nodes, proceed to the next section.
 
+``` bash
+make install
+```
+
+This step will copy the final ArgoDSM include files to `/usr/local/include` and
+libraries to `/usr/local/lib/`. You can choose a different path above if you
+want, but remember to either set `INCLUDE_PATH`, `LIBRARY_PATH`, and
+`LD_LIBRARY_PATH` accordingly, or provide the correct paths (`-L`, `-I`, and
```

Comment:
The `-L` and `-I` flags should swap places to match the list of environment variables. (Or the environment variables should change order.)